core/print_api.php

```diff
@@ -1877,8 +1878,9 @@ function print_bug_attachment_header( array $p_attachment ) {
 	}
 
 	if( $p_attachment['can_delete'] ) {
+		$t_delete_token = $p_bug_file_delete_token ? $p_bug_file_delete_token : form_security_param( 'bug_file_delete' );
```

Comment:
I think it's cleaner to pass only the token id, not the prebuilt string.
However, you'll need to add the option to form_security_param(), to accept the token id as parameter. Like it's already done in form_security_field()